src/librsync.h

```diff
@@ -37,9 +37,36 @@
 #define _RSYNC_H
 
 #include <sys/types.h>
-#include "types.h"
 #include "librsync-config.h"
 
+#define HAVE_UINT64 1 /* assume success, but might be undefined below */
+
+#if defined(HAVE_STDINT_H)
+#  include <stdint.h>
```

Comment:
I wonder if there is much point in supporting systems without stdint.h when blake2.h and rollsum.h both depend on it.

Is it possible for librsync.h and supporting libs to be compiled on a system with stdint.h and then deployed on a system without it? If so, then I guess it is worth it, but otherwise it makes things much simpler if we just require it.